lib/quake_parser.ex

```diff
@@ -3,14 +3,31 @@ defmodule QuakeParser do
   `QuakeParser` is a parser for Quake game logs.
   """
 
-  alias QuakeParser.{Game, FindPlayers, KillsOfPlayer}
+  alias QuakeParser.{Game, FindPlayers, KillsOfPlayer, DeathReport}
 
+  @doc """
+  Parse the Quake log file, and return a QuakeParser.Game object for each game.
+  """
   def start(path) do
     validate_file(path)
```

Comment:
```suggestion
    with {:ok, file} <- File.read(path),
    games <- find_games(file) do
      Enum.map(games, &parse_game/1)
 end
```

Acho que ficaria melhor, assim o error do File.open vai pra cima e o usuário pode tratar